contracts/test/userLeave.js

```diff
@@ -126,7 +126,10 @@ contract('PoolDeposits', accounts => {
     await poolDeposits.deposit(mintAmount, { from: accounts[1] });
 
     // withdraw their funds
```

Comment:
weird behaviour, if I try to do 2 event tests in a single test, get an error throw, can't find filter etc... Worked around this and tested different events in different tests.

Reply:
And commenting out the `expectEvent.inLogs` fixes it?

Reply:
This is fixed please ignore. Open Zeppelin has updated there events testing and it was deprecated the way I was doing it. All fixed now

Reply:
This is fixed. Method I was using was deprecated